frontend/src/components/personal/EmailChangeForm.test.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react'
+import { render, screen, waitFor} from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import { Provider } from 'react-redux'
+import '@testing-library/jest-dom'
+
+import {store} from '../../index'
+import EmailChangeForm from './EmailChangeForm'
+
+describe('EmailChangeForm', () => {
+  it('title in place', () => {
+    render(<Provider store={store}><EmailChangeForm /></Provider>)
+    expect(screen.getByText(/CHANGE EMAIL/i)).toBeInTheDocument()
+
+  })
+  
+  it('Formik validation fails if new email is not an email', async () => {
+    render(<Provider store={store}><EmailChangeForm /></Provider>)
+    userEvent.type(screen.getByLabelText(/email/i), '1234')
+     
+    userEvent.click(screen.getByTestId(/sbmtEmail/i))
```

Comment:
`getByTestId` is a last option to get an element. Try to use roles or labels.

Reply:
corrected